src/query/sql/src/planner/binder/project.rs

```diff
@@ -290,3 +308,75 @@ impl<'a> Binder {
         Ok(output)
     }
 }
+
+// Pre-check exclude_col is legal
+fn precheck_exclude_cols(
+    input_context: &BindContext,
+    exclude_cols: &HashSet<String>,
+    db_name: Option<&Identifier>,
+    table_name: Option<&Identifier>,
+) -> Result<()> {
+    let all_columns_bind = input_context.all_column_bindings();
+    let mut qualified_cols_name: HashMap<String, u8> = HashMap::new();
+
+    fn fill_qualified_cols(
+        qualified_cols_name: &mut HashMap<String, u8>,
```

Comment:
use `HashSet`

Reply:
Truly, the value is no used. HashSet is better